R/data/data_validation.R

```diff
@@ -45,11 +45,6 @@ ValidateUploadedData <- function(data, outcome_type) {
     return(result)
   }
 
-  result <- .ValidateSingleArmStudies(data)
-  if (!result$valid) {
-    return(result)
-  }
-
```

Comment:
Everything missing from this file should be reinstated.

Reply:
Done :)